### PR TITLE
copy changes of 1.x to 2.x

### PR DESCRIPTION
**Pull request by gharlan**
- updated travis.yml
- updated composer.json
- changed namespace to "Propel\Bundle\PropelBundle"

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,38 +1,35 @@
...
 env:
-    - SYMFONY_VERSION="~2.5.0"
-    - SYMFONY_VERSION="~2.6.0"
-    - SYMFONY_VERSION="~2.7.0"
-    - SYMFONY_VERSION="~2.8.0"
-    - SYMFONY_VERSION="~3.0.0"
-    - SYMFONY_VERSION="dev-master"
+    - COMPOSER_FLAGS="--prefer-lowest"
+    - SYMFONY_VERSION="^2.8.2"
+    - SYMFONY_VERSION="^2.8.2" COMPOSER_FLAGS="--prefer-lowest"
```
Comment (havvg) [inline .travis.yml:18]: The 2.0 branch should be tested on Symfony 3.0, too.

File: DataFixtures/Loader/AbstractDataLoader.php [hunk 2]
```diff
@@ -8,10 +8,10 @@
  * @license    MIT License
  */
 
-namespace Propel\PropelBundle\DataFixtures\Loader;
+namespace Propel\Bundle\PropelBundle\DataFixtures\Loader;
 
-use Propel\PropelBundle\DataFixtures\AbstractDataHandler;
-use Propel\PropelBundle\Util\PropelInflector;
+use \Propel\Bundle\PropelBundle\DataFixtures\AbstractDataHandler;
+use \Propel\Bundle\PropelBundle\Util\PropelInflector;
```
Comment (havvg) [inline DataFixtures/Loader/AbstractDataLoader.php]: Two `\` sneaked in.

File: .travis.yml (hunk 1 above)
Comment (gharlan) [inline .travis.yml:18]: Should I change last line, or add new one?

**Comment by havvg** [comment]
You missed the `Resources/acl_schema.xml`.

File: .travis.yml (hunk 1 above)
Comment (havvg) [inline .travis.yml:18]: Add the two for `^3.0.0` (one with and without `--prefer-lowest`).

**Comment by havvg** [comment]
Just read the diff, you got the schema, but Travis fails.

File: Tests/Fixtures/Model/Base/BookQuery.php [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
+namespace Propel\Bundle\PropelBundle\Tests\Fixtures\Model\Base;
 
 use \Exception;
 use \PDO;
-use Propel\PropelBundle\Tests\Fixtures\Model\Book as ChildBook;
-use Propel\PropelBundle\Tests\Fixtures\Model\BookQuery as ChildBookQuery;
-use Propel\PropelBundle\Tests\Fixtures\Model\Map\BookTableMap;
+use \Propel\Bundle\PropelBundle\Tests\Fixtures\Model\Book as ChildBook;
+use \Propel\Bundle\PropelBundle\Tests\Fixtures\Model\BookQuery as ChildBookQuery;
+use \Propel\Bundle\PropelBundle\Tests\Fixtures\Model\Map\BookTableMap;
```
Comment (havvg) [inline Tests/Fixtures/Model/Base/BookQuery.php]: There are some more (`\`) hidden among the files.

**Comment by gharlan** [comment]
I had to change the required propel version in composer.json to "dev-master". Otherwise I got 2.0.0-alpha1 when testing with `--prefer-lowest`.
Is it ok, or is there a better solution?

File: .travis.yml (hunk 1 above)
Comment (gharlan) [inline .travis.yml:18]: done

**Comment by gharlan** [comment]
The remaining failures on travis are unrelated to the changes of this PR. They are introduced by this change: https://github.com/propelorm/Propel2/pull/1097

**Comment by havvg** [comment]
Are you sure on the remaining failures? The build passes on `2.0` without this pull request. What am I missing?

**Comment by gharlan** [comment]
The build was done before the change was made. Or did you rebuild?

**Comment by havvg** [comment]
Ah, now I get it, the changes in Propel make the bundle fail on its current version. Building bundle 2.0 now.

**Comment by havvg** [comment]
`dev-master` on Propel2 is fine by me in the current situation.

**Comment by havvg** [comment]
Looks good to me, anything left to do?

**Comment by gharlan** [comment]
from my side it is ready.

**Comment by havvg** [comment]
Thank you very much!

**Pull request merged**